src/utils/segmenter/segmenter.py

```diff
@@ -1,50 +1,30 @@
-from rasterio.transform import xy
-from shapely.geometry import Polygon
-
 from .area_computer import build as area_build, rebuild as area_rebuild
-from .loop_computer import build as loop_build, rebuild as loop_rebuild
+from .boundary_computer import\
+    build as boundary_build, \
+    rebuild as boundary_rebuild
 from .segment_computer import build as segment_build, update as segment_update
 
 
 class Segmenter:
-    def __init__(self, data, transform):
-        self.data = data
-        self.transform = transform
-        self.build()
-
-    def build(self):
-        self.areas = area_build(self.data, self.transform)
-
-        # Add a polygon that is the perimeter of data.
-        # This ensures that we fix the boundary of the data in place.
-        self.perimeter = self.build_perimeter()
-        self.loops = loop_build(self.perimeter, self.areas)
-
-        self.segments = segment_build(self.loops)
-
-    def build_perimeter(self):
```

Comment:
We don't need this since it doesn't solve the problem it was meant to solve. I'm getting rid of this now because it doesn't make sense with the clarified purpose of `Segmenter`.